### PR TITLE
Fix utilization values mismatch queries

### PR DESCRIPTION
**Pull request by aruniiird**
Fixed latency query and removed the previous (not-needed) fix.

### CODE REVIEW

**Comment by agarwal-mudit** [comment]
@aruniiird Do we have a BZ for this PR? What is the priority?
/retest

**Comment by bipuladh** [comment]
/retitle Bug 2026144: Fix utilization values mismatch queries

**Comment by agarwal-mudit** [comment]
Fix is required for https://bugzilla.redhat.com/show_bug.cgi?id=2026144
However, the bug number is not required in the title for main branch fixes.

File: metrics/mixin/rules/rules.libsonnet [hunk 1]
```diff
@@ -36,7 +36,6 @@
       },
       {
         name: 'ODF_standardized_metrics.rules',
-        interval: '30s',
```
Comment (umangachapagain) [inline metrics/mixin/rules/rules.libsonnet:39]: Why do we need to remove this interval?

File: metrics/mixin/rules/rules.libsonnet [hunk 2]
```diff
@@ -91,7 +90,7 @@
           {
             record: 'odf_system_latency_seconds',
             expr: |||
-              avg by (namespace, managedBy, job, service)
+              sum by (namespace, managedBy, job, service)
```
Comment (umangachapagain) [inline metrics/mixin/rules/rules.libsonnet:93]: Can you explain why changing "avg" to "sum" and removing division by 2 works here?
Seems completely different from original query.

File: metrics/mixin/rules/rules.libsonnet (hunk 1 above)
Comment (aruniiird) [reply metrics/mixin/rules/rules.libsonnet:39]: The BZ for which this PR is initiated is actually a re-opened BZ.
The line which is removed here was the previous fix which was not effective (and QE had to re-open the BZ again) and it is not needed now

**Review by umangachapagain — APPROVED** [review]

**Pull request merged**